manifests/init.pp

```diff
@@ -120,7 +120,7 @@
   $ca_key = "${certs::pki_dir}/private/${default_ca_name}.key"
   $ca_cert = "${certs::pki_dir}/certs/${default_ca_name}.crt"
   $ca_cert_stripped = "${certs::pki_dir}/certs/${default_ca_name}-stripped.crt"
-  $ca_key_password = cache_data('ca_key_password', generate_password())
+  $ca_key_password = cache_data('ca_key_password', random_password(8))
```

Comment:
Nitpicky, but 8 is very short.  I'd say 24 at least.  No one's going to be typing this in